src/index.html

```diff
@@ -1,17 +1,141 @@
+            autocomplete="off"
+            required
+        /></label>
+
+        <label class="box"
+          >How old are You?:<input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="22"
+            autocomplete="off"
+            required
+        /></label>
+
+        <label class="box"
+          >Full date of birth:<input type="date" id="date" name="date"
+        /></label>
+        <label class="box"
+          >I accept the term of the agreement<input type="checkbox" />
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="box"
+          >Email:<input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+        /></label>
+
+        <label class="box"
+          >Password:<input
+            type="password"
+            id="password"
+            name="password"
+            autocomplete="off"
+            required
+        /></label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <div class="box">
+          Do you love cats?
+
+          <label><input type="radio" name="loveCats" value="yes" /> Yes</label>
+
+          <label><input type="radio" name="loveCats" value="no" /> No</label>
+        </div>
+
+        <label class="box"
+          >What are you favorite color?:<input type="color" name="color"
+        /></label>
+
+        <label class="box"
+          >What time dou you go to bed?:<input
+            type="time"
+            id="time"
+            name="time"
+        /></label>
+        <label class="box"
+          >What are your favorite brand of cars?:<select
+            id="level"
+            name="level"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select></label
+        >
+
+        <label class="box"
+          >How do you rate our work:<input
+            type="range"
+            id="range"
+            name="range"
+            min="0"
+            max="10"
+        /></label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+        <label class="box"
+          >Comments:<textarea
+            placeholder="Tell us about yourself"
+            minlength="10"
+            maxlength="500"
+            required
+          ></textarea>
+        </label>
+        <label class="box"
+          >Would you recommend us?:<select>
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select></label
+        >
+      </fieldset>
+      <button type="submit" class="container">Submit</button>
+    </form>
+
```

Comment:
redundant empty line between parent and child tags

Reply:
Good evening.
Could you check the latest fix?